tendrl/node_agent/manager/tendrl_definitions_node_agent.py

```diff
@@ -30,6 +30,36 @@
           type: String
       enabled: true
       value: nodes/$Node_context.node_id/Memory
+    Disk:
+      attrs:
+        device_name:
+          type: String
+        fs_type:
+          type: String
+        fs_uuid:
+          type: String
+        model:
+          type: String
+        mount_point:
+          type: List
+        name:
+          type: String
+        parent:
+          type: String
+        size:
+          type: Integer
+        disk_type:
+          type: String
+        used:
+          type: Boolean
+        ssd:
+          type: Boolean
+        vendor:
+          type: String
+        disk_id:
+          type: String
+      enabled: true
+      value: nodes/$Node_context.node_id/Disks/device_name
```

Comment:
In file https://github.com/Tendrl/node_agent/pull/64/files#diff-ce609b2cc94f6f7d7cd1d48f43d75492 you have defined disk etcd layout as "nodes/%s/Disks/%s/%s" 

and here it is "nodes/$Node_context.node_id/Disks/device_name", which one is correct?